### DISCUSSION TITLE
Allow Difficulty Adjustment AR Changes to take effect after Double Time

### DISCUSSION THREAD

**Discussion by iamtechi27** [ppy/osu#14170] [Ideas] [answered]
Currently, I have to think about what approach rate to set if I want to use Double Time customized to something like 1.2x if I want to play at a particular approach rate. It would be nice if there was a checkbox or something to have the customized approach rate take effect after the double time mod. Or even just a checkbox that makes the game do the math for me, since I know double time approach rate works a little differently to normal approach rate.

**Comment by gagahpangeran** [comment]
https://github.com/ppy/osu/issues/10580

**Reply by iamtechi27** [reply]
That's part of it. That thread refers to displaying the resulting AR, not setting it.

**Comment by smoogipoo** [comment, marked as answer]
Broadly speaking, we've tried to avoid inter-mod interactions like this.

Since DT affects clock time and _not_ AR directly, this closely relates to https://github.com/ppy/osu/discussions/13187.
